cloud/multi-factor-authentication.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Multi-factor user authentication
+excerpt: Manage Multi-factor user authentication for your Timescale Cloud account
+product: cloud
+keywords: [mfa, accounts, admin]
+tags: [two-factor user authentication]
+---
+
+## Multi-factor user authentication
+
+You can use two-factor authentication also known as 2-Step verification or 2FA
+to log in to your Timescale Cloud account. This enables secure logins that
+requires a second authentication code in addition to the user password. You
+require an authentication code, provided by the Google Authenticator app on
+your mobile device.
+
+### Before you begin:
+
+*   Install the [Google Authenticator application][install-google-authenticator]
+  on your mobile device.
+
+<procedure>
+
+### Configuring two-factor authentication
+
+1.  [Log in to your Timescale Cloud account][cloud-login].
+1.  Click the `User name` icon in the top-right of the console and select `Account`.
+1.  In the `Account` page, click `Add two-factor authentication` button.
+1.  On your mobile device, open the Google Authenticator app, tap `+` and select
+    `Scan a QR code`.
+1.  On your mobile device, scan the QR code provided by Timescale Cloud in the
+    `Connect to an authenticator app` dialog and click `Next`.
+1.  Type the verification code provided by the Google Authenticator app, and click
+    `Next`.
+1.  In the `Save your recovery codes` dialog, you can copy, download, or print the
+    recovery codes that appear.
+1.  Click `ok, i saved my recovery codes` after you have saved the recovery codes.
+    <highlight type="important">
+    A recovery code is a backup way to log in to Timescale Cloud when you can't
+    get your verification code from  your authenticator app. If you enable
+    two-factor authentication, save your recovery codes.
+    </highlight>
```

Comment:
I'm not sure we need this as an admonition, we can just add it in to the step about saving ☝️